### PR TITLE
Add the log handler

### PR DESCRIPTION
**Pull request by slacAWallace**
You may need to reactivate to license the TCP/IP module, and add the TCP/IP library.

@klauer

### CODE REVIEW

File: plc-lfe-motion/lfe_motion/POUs/PRG_1_PlcTask.TcPOU [hunk 1]
```diff
@@ -6,9 +6,13 @@
...
     Only "fast" programs are allowed here,
     anything else needs to be relegated to a second task
-*)]]></Declaration>
+*)
+
+fbLogHandler : FB_LogHandler;
+]]></Declaration>
     <Implementation>
-      <ST><![CDATA[PRG_2_PMPS_PRE();
+      <ST><![CDATA[
```
Comment (ZLLentz) [inline plc-lfe-motion/lfe_motion/POUs/PRG_1_PlcTask.TcPOU]: Why did you add a newline at the start of the file?

File: plc-lfe-motion/lfe_motion/POUs/PRG_1_PlcTask.TcPOU [hunk 2]
```diff
@@ -21,7 +25,10 @@ PRG_SL1L0_POWER();
 PRG_SL2L0_POWER();
 PRG_SP1L0_KMONO();
 
-PRG_3_PMPS_POST();]]></ST>
+PRG_3_PMPS_POST();
+
+fbLogHandler();
+]]></ST>
```
Comment (ZLLentz) [inline plc-lfe-motion/lfe_motion/POUs/PRG_1_PlcTask.TcPOU]: And at the end?

File: plc-lfe-motion/lfe_motion/POUs/PRG_1_PlcTask.TcPOU (hunk 1 above)
Comment (slacAWallace) [reply plc-lfe-motion/lfe_motion/POUs/PRG_1_PlcTask.TcPOU]: O_O

I started adding the handler there, then just left the newline because it's slightly easier to read on github.

Comment (ZLLentz) [reply plc-lfe-motion/lfe_motion/POUs/PRG_1_PlcTask.TcPOU]: :man_shrugging:

File: plc-lfe-motion/lfe_motion/POUs/PRG_1_PlcTask.TcPOU (hunk 2 above)
Comment (slacAWallace) [reply plc-lfe-motion/lfe_motion/POUs/PRG_1_PlcTask.TcPOU]: Yep. Same deal.

**Comment by ZLLentz** [comment]
You shouldn't bother making nonfunctional whitespace changes because they are inevitably going to be changed back by accident later

**Comment by slacAWallace** [comment]
Fair point and noted. :)

**Comment by klauer** [comment]
https://pcdshub.github.io/lcls-plc-lfe-motion/plc-lfe-motion.tsproj.html#pou-prg-1-plctask

**Comment by ZLLentz** [comment]
The plc project does not build after merging this PR. I should have checked more carefully to make sure you tried to build before pushing.

**Comment by ZLLentz** [comment]
One more follow-up: the checked out plc project was no longer possible to run even after reverting this PR. It was seemingly impossible to satisfy a license requirement for empty string, and the tmc file would refuse to build. I had to check out again from master to get it to work.

**Comment by klauer** [comment]
That's pretty concerning regarding the licenses/global type stuff. I hope Nate can shine some light on better practices.

As far as the syntax errors - big oops. Building the project on CI would be ideal to catch anything and everything that could go wrong, but I think we're a long ways off from that. 

In the meantime, I'm hoping my weekend side-project blark will be able to catch some of these (even if its grammar specification is by no means perfect). Once I get around to some fixes regarding comments, I'll introduce an allowed-failure CI step that parses the source code with it.  For what it's worth, it does catch this specific mistake:

```
[/Users/klauer/docs/Repos/lcls-plc-lfe-motion/plc-lfe-motion/lfe_motion/POUs/PRG_1_PlcTask.TcPOU] PRG_1_PlcTask
---------------------------------------------------------------------------------------------------------------
(UnexpectedCharacters) No terminal defined for ':' at line 8 col 14

fbLogHandler : FB_LogHandler;
             ^
```
... but also fails elsewhere where it shouldn't.
